basestar-storage/src/test/java/io/basestar/storage/TestStorage.java

```diff
@@ -1406,4 +1410,45 @@ public void testSqlQuery() throws Exception {
                 "state", "District of Columbia"
         )), firstPage.get(0));
     }
+
+    @Test
+    public void testSqlQueryNull() throws Exception {
+
+        assumeTrue(supportsSql());
+
+        final Storage storage = storage(namespace);
+
+        bulkLoad(storage, loadAddresses());
+
+        final QuerySchema schema = namespace.requireQuerySchema(SQL_QUERY);
+
+        final Page<Map<String, Object>> firstPage = storage.query(Consistency.ATOMIC, schema, Immutable.map("city", null), Constant.TRUE, Immutable.list(), ImmutableSet.of())
+                .page(10).get();
+
+        assertEquals(13, firstPage.size());
+    }
+
+    @Test
+    public void testDefaultSequence() {
+
+        final Storage storage = storage(namespace);
+        final SequenceSchema schema = namespace.requireSequenceSchema(DEFAULT_SEQUENCE);
+        assumeTrue(storage.storageTraits(schema).supportsSequence());
+
+        assertEquals("0", storage.increment(schema).thenApply(schema::format).join());
+        assertEquals("1", storage.increment(schema).thenApply(schema::format).join());
+        assertEquals("2", storage.increment(schema).thenApply(schema::format).join());
+    }
+
+    @Test
+    public void testCustomSequence() {
```

Comment:
Storage specification test for a sequence with custom start and format expression